qase-javascript-commons/src/models/test-result.ts

```diff
@@ -23,7 +22,7 @@ export type TestResultType = {
   title: string
   signature: string
   run_id: number | null
-  testops_id: number | null
+  testops_id: number | number[] | null
```

Comment:
I could not find the place where an array of strings is converted into an array of ints.

```
qase.id(['1', '2'])
```

Reply:
We do it in the reporters. For example, https://github.com/qase-tms/qase-javascript/blob/v2-beta/qase-cypress/src/reporter.ts#L55